packages/react/src/components/form/Switch/Switch.module.css

```diff
@@ -0,0 +1,170 @@
+.switch {
+  --fds-inner-focus-border-color: var(--fds-semantic-border-focus-boxshadow);
+  --fds-outer-focus-border-color: var(--fds-semantic-border-focus-outline);
+  --fds-focus-border-width: 3px;
+  --fds-transition: 200ms;
+
+  min-width: 44px;
+  min-height: 44px;
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto 1fr;
+  grid-template-areas:
+    'input label'
+    '. description';
+  gap: 0 var(--fds-spacing-2);
+  grid-auto-flow: column;
+}
```

Comment:
Her er både `min-width` og `gap` konstant. Det gjør at mellomrommet mellom switchen og teksten blir større når man nedjusterer størrelsen. Om noe burde det vel bli mindre? Regner med at dette er for å tilfredsstille krav om klikkflatestørrelse, men her så går det vel greit å gjøre den smalere siden teksten også kan klikkes på?

Reply:
`gap` bruker spacing tokens (på kolonne-nivå), som er fluid. Den bruker `rem` og `vw` så den blir mindre, jo mindre skjermen er og størrelsen på komponenten. WCAG kravet er at [trykkflate](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html) for "control" må være 44x44, og brukertester viser at folk flest trykker på "control" framfor labelen. 

Ser dog at jeg har en padding-left på label som ikke skal være der. I fix! 🤪

Reply:
> Den bruker rem og vw så den blir mindre, jo mindre skjermen er og størrelsen på komponenten.
Følger den `size`-propen også?

> WCAG kravet er at [trykkflate](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html) for "control" må være 44x44, og brukertester viser at folk flest trykker på "control" framfor labelen.
Selve kontrollen (ikonet) er jo uansett mindre enn 44x44 ... Slik jeg forstår det er dette to forskjellige ting; den visuelle kontrollen (kun ikon) og den totale trykkflaten (ikon + label + området rundt).

Reply:
> > Den bruker rem og vw så den blir mindre, jo mindre skjermen er og størrelsen på komponenten.
> 
> Følger den `size`-propen også?
> 
Jepp! Derfor vi vi har `<Paragraph>` som ytterst element. Vi sendere `size` til den som justerer font-størrelsen. Ikonet har da høyden satt til em, slik at den visuelt har korrekt størrelse i henhold til font-størrelsen.

Ser og en liten visuel bug med hengelås-ikonet nå da ikonene til Aksel er satt til `rem` og ikke `em`. Fikser det! 

> > WCAG kravet er at [trykkflate](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html) for "control" må være 44x44, og brukertester viser at folk flest trykker på "control" framfor labelen.
> 
> Selve kontrollen (ikonet) er jo uansett mindre enn 44x44 ... Slik jeg forstår det er dette to forskjellige ting; den visuelle kontrollen (kun ikon) og den totale trykkflaten (ikon + label + området rundt).

Ja, kravet er trykkflaten skal være 44x44, det visuelle kan være mindre men du skal gi en indikasjon mtp. at du er i "kontakt" med komponenten. Dette gjøres vi med `cursor: pointer` og `:hover` bakgrunnsfarge. Vi har løst det slikt på Radio og Checkbox, men unnlatt Switch hover bakgrunnsfargen ser jeg. Det er nok et designvalg her da switcher har vel ikke typisk bakgrunnsfarger ved hover? 🤔